pack/msm_encounter.json

```diff
@@ -0,0 +1,79 @@
+[
+  {
+    "boost": 2,
+    "code": "05025",
+    "faction_code": "encounter",
+    "name": "Home by Dawn",
+    "pack_code": "msm",
+    "position": 25,
+    "quantity": 1,
+    "set_code": "ms_marvel",
+    "text": "<b><i>Give to the Kamala Khan player</i></b>\nYou may flip to your alter-ego form. Choose:\n- Exhaust Kamala Khan → remove Home by Dawn from the game.\n- Discard 1 <b><i>PERSONA</i></b> support you control. If no support was discarded this way, this card gains surge. Discard this obligation.",
```

Comment:
`[[Persona]]` for the special trait text.

Reply:
Updated a new commit